dns/unbound-plus/src/opnsense/mvc/app/models/OPNsense/Unboundplus/Miscellaneous.xml

```diff
@@ -1,14 +1,14 @@
 <model>
     <mount>//OPNsense/unboundplus/miscellaneous</mount>
     <description>Unbound Miscellaneous configuration</description>
-    <version>0.0.2</version>
+    <version>0.0.3</version>
     <items>
         <privatedomain type="CSVListField">
             <Required>N</Required>
         </privatedomain>
         <dotservers type="CSVListField">
             <Required>N</Required>
-            <mask>/^[a-fA-F0-9\.\@]{1,46}$/</mask>
+            <mask>/^[a-fA-F0-9\.\,\:\@]{1,512}$/</mask>
```

Comment:
@AdSchellevis isn't there a more graceful way... csv list field needs comma as valid char validator?

Reply:
I was thinking about using NetworkField combined with asList, but this would disallow setting the port. I also didn't want to make the port always using static 853 (you never know), but I also didn't want to make an extra grid with adding IP/port pairs, so CSVListField was the smoothest approach.

Reply:
I think you can set the separator  in this type, but to be honest, I'm not a huge fan of the csv construct, it was one of the first list types which is rather free-form.  Do we have a list of what should be allowed? (when merging patches here, we have to make sure we copy the same to core/master as well during the transition phase by the way)

Reply:
If we agree that port 853 is static I can make it a NetworkField and add 853 in templating. 
Otherwise I'd go on with CSV